src/langgraph-platform/hybrid.mdx

```diff
@@ -34,4 +34,50 @@ For information on how to deploy a [LangGraph Server](/langgraph-platform/langgr
 
 <Tip>
   If you would like to deploy to Kubernetes, you can follow the [Hybrid deployment guide](/langgraph-platform/deploy-hybrid).
-</Tip>
\ No newline at end of file
+</Tip>
+
+## Listeners
+
+In a hybrid deployment, one or more ["listener" applications](/langgraph-platform/data-plane#”listener”-application) can be deployed depending on the organization of LangSmith workspaces and Kubernetes clusters.
+
+**Kubernetes cluster organization**
+- One or more listeners can be deployed on a Kubernetes cluster.
+- A listener can be configured to deploy to one or more Kubernetes namespaces in the cluster.
+- It's the responsibility of the owner of Kubernetes to decide the optimal organization of listeners for their use case. This requires planning ahead of time for how LangGraph Server deployments should be organized.
+
+**LangSmith workspace organization**
+- A workspace can have one or more listeners associated with it.
```

Comment:
Shall we link to the LS workspace docs here? Maybe: https://docs.smith.langchain.com/administration/how_to_guides/organization_management/set_up_workspace#create-a-workspace?

Reply:
I think it's fine to omit this link, but what's our general linking strategy between LangSmith and LangGraph Platform?

In other places in the LangGraph Platform docs, I've linked to LangSmith for specific things, but I don't follow any particular rules.